js/app.js

```diff
@@ -0,0 +1,44 @@
+$(document).ready(function() { 
+
+
+	//Hide 'Read More' text and 'Learn More' text initially
+ 	 $('#show-this-on-click', '#learnmoretext').hide ();
+
+ 	// setup the click handlers for 'Read more' and 'Learn More'
+ 	 $('#readMore').click('#show-this-on-click');
+  	 $('#learnMore').click('#lxearnmoretext');
+
+  	function readMore (event) {
+            //Show Blog Post Content on 'Read More' click, Display Blog Post Content sliding down on click, and display 'Read Less' text on 'Read More' Click
+            event.preventDefault();
+            $('#posts .readmore').hide();
+            $('#show-this-on-click').slideDown();
+            $('#posts .readLess').show();
```

Comment:
Careful, its written `.readless`, not `.readLess`. That's why the `.readless` link never appears.